tests/parse_config_update_test.py

```diff
@@ -170,3 +173,53 @@ def test_parse_streams_skips_stream_info_if_remove_config_and_schema_present_but
     config_message = deserialise_rf5k(message)
     streams = tuple(_parse_streams(CommandType.REMOVE, config_message.streams))
     assert not streams
+
+
+@patch(
+    "forwarder.parse_config_update.deserialise_rf5k",
+    side_effect=RuntimeError("Runtime Error"),
+)
+def test_parse_config_update_when_runtime_error_occurs(mock_func):
+    test_channel_name = "test_channel"
```

Comment:
This doesn't need to a variable. The string can be put directly into StreamInfo

Reply:
Yeah, never copy-paste code. ;) Fixed.